assets/locales/en.json

```diff
@@ -2,7 +2,7 @@
   "daoRoundData": {
     "fundingRound": "Funding Round",
     "amountProposals": "Amount proposals",
-    "countdown": "Countdown",
+    "countdown": "Voting ends in",
```

Comment:
praise: good idea to keep all information inside the title. Thereby we can just display the value inside the value field.